examples/svm/plot_rbf_parameters.py

```diff
@@ -53,13 +53,18 @@
 each point correctly (larger ``C`` values) hence the diagonal of good
 performing models.
 
-Finally one can also observe that for some intermediate values of ``gamma`` we
-get equally performing models when ``C`` becomes very large: it is not
-necessary to regularize by enforcing a larger margin. The radius of the RBF
-kernel alone acts as a good structural regularizer. In practice though it
-might still be interesting to simplify the decision function with a lower
-value of ``C`` so as to favor models that use less memory and that are faster
-to predict.
+Finally, one can also observe that for some intermediate values of ``gamma`` we
+get equally performing models when ``C`` becomes very large. This suggests that
+the support vectors do not change anymore. The radius of the RBF kernel alone
+acts as a good structural regularizer. Increasing ``C`` further doesn't help,
+likely because there are no more training points in violation (inside the
+margin or wrongly classified), or at least no better solution can be found.
+Performance being equal, it makes sense to use the smaller ``C`` values, since
```

Comment:
Since "performance" can also suggest "computation time"

Also "may"  because of the tradeoff mentioned later

```suggestion
Scores being equal, it may make sense to use the smaller ``C`` values, since
```